features/leeftijd_bepaling.feature

```diff
@@ -1,57 +1,188 @@
-	| 31 mei 2019     | <niet aanwezig>   | In de geboorte maand weten we niet wanneer de persoon jarig is                         |
-	| 01 juni 2019    | 36                | Na de geboorte maand weten we zeker dat de persoon 1 jaar ouder is geworden            |
-	| 30 april 2019   | 35                | Voor de geboorte maand weten we zeker dat de persoon nog niet 1 jaar ouder is geworden |
-
-Scenario: Alleen jaar van geboorte datum is bekend
-	Gegeven een persoon met geboortedatum mei 1983
-	Als de persoon op <raadpleeg datum> wordt geraadpleegd
-	Dan is attribuut leeftijd niet aanwezig
-
-Scenario: Persoon is overleden
-	Gegeven de persoon met burgerservicenummer "999993197" is overleden op 28-04-2019
-	Als de  persoon met burgerservicenummer 999993197 wordt geraadpleegd met fields=leeftijd
-	Dan is attribuut leeftijd niet aanwezig
-
-Scenario: leeftijd wordt wel geleverd bij een overleden kind (omdat alleen gegevens van de persoonslijst van de gevraagde persoon worden gebruikt)
-	Gegeven de persoon met burgerservicenummer "999991280" heeft een kind met burgerservicenummer "999993197"
-	En de persoon met burgerservicenummer "999993197" is overleden op 28-04-2019
-	Als de  persoon met burgerservicenummer 999994268 wordt geraadpleegd met fields=kinderen
-	Dan heeft het kind met burgerservicenummer "999993197" een attribuut "leeftijd" met een waarde
-
-Abstract Scenario: Geboren op 29 februari in een schrikkeljaar
-	Gegeven een persoon met 29 februari 1996
-	Als de persoon op <raadpleeg datum> wordt geraadpleegd
-	Dan heeft attribuut leeftijd de waarde <leeftijd>
-
-	Voorbeelden:
-	| raadpleeg datum  | leeftijd |
-	| 29 februari 2016 | 20       |
-	| 28 februari 2017 | 20       |
-	| 01 maart 2017    | 21       |
+	Rule: bij een volledig bekende geboortedatum wordt de leeftijd in jaren geleverd
+		
+		Abstract Scenario: Volledig geboortedatum
+			Gegeven het systeem heeft een persoon met de volgende gegevens
+			| naam                | waarde    |
+			| burgerservicenummer | 555550001 |
+			En de persoon heeft de volgende 'geboorte' gegevens
+			| naam  | waarde   |
+			| datum | 19830526 |
+			Als personen op '<raadpleeg datum>' wordt gezocht met de volgende parameters
+			| naam                | waarde                          |
+			| type                | RaadpleegMetBurgerservicenummer |
+			| burgerservicenummer | 555550001                       |
+			| fields              | burgerservicenummer,leeftijd    |
+			Dan heeft de persoon met burgerservicenummer '555550001' alleen de volgende gegevens
+      | burgerservicenummer | leeftijd   |
+      | 555550001           | <leeftijd> |
+
+			Voorbeelden:
+			| raadpleeg datum  | leeftijd |
+			| 1 januari 2019   | 35       |
+			| 25 mei 2019      | 35       |
+			| 26 mei 2019      | 36       |
+			| 30 november 2019 | 36       |
+			| 10 januari 2020  | 36       |
+
+		Abstract Scenario: Volledig geboortedatum van een kind
+			Gegeven het systeem heeft een persoon met de volgende gegevens
```

Comment:
Is dit scenario wel nodig ? 
Het berekenen van de leeftijd van een kind is toch niet anders dan die van een persoon ? De feature gaat om het berekenen van de leeftijd.

Reply:
Leuk mij nuttig te tonen dat dit ook bij een kind moet worden gedaan

Reply:
ik voeg ook een extra kind toe, zodat zichtbaar is dat op één persoon met meerdere kinderen elk kind een leeftijd berekend krijgt